query-engine/connector-test-kit/src/test/scala/writes/unchecked_writes/UncheckedCreateSpec.scala

```diff
@@ -0,0 +1,385 @@
+package writes.unchecked_writes
+
+import org.scalatest.{FlatSpec, Matchers}
+import util._
+
+// Important: This test covers ALL top level create inputs, like create & upsert,
+// because schema building uses the exact same types under the hood.
```

Comment:
Not sure what you mean with exact types. Does that refer to stuff that gets added implicitly to a Prisma schema?

Reply:
It means exactly what it says: An `upsertOne` uses the _exact_ create type used for `createOne`, for example, which makes testing upsert for the create path irrelevant.